Makefile

```diff
@@ -2,8 +2,8 @@
 # Setup Makefile to match your environment
 #
 PYTHON=/usr/bin/python3
-PYFLAKES=/usr/bin/pyflakes-3
-PYCODESTYLE=/usr/bin/pycodestyle-3
+PYFLAKES=/usr/bin/pyflakes
+PYCODESTYLE=/usr/bin/pycodestyle
```

Comment:
This breaks it for everyone else. If openSUSE is not providing versioned names, then that's a breakage on their side.

Reply:
There are versioned names, but they also include the minor version number: `/usr/bin/pyflakes-3.7`

Reply:
if there is no common naming schema, should the existence of the programs be checked? Or add a distro specific section?